phpBB/includes/acp/acp_styles.php

```diff
@@ -756,22 +756,26 @@ protected function find_available($all)
 				// Style is already installed
 				continue;
 			}
-			$cfg = $this->read_style_cfg($dir);
-			if ($cfg === false)
+
+			try
+			{
+				$style_data = $this->read_style_composer_file($dir);
+			}
+			catch (\DomainException $e)
 			{
-				// Invalid style.cfg
+				// Invalid composer.json
 				continue;
 			}
 
 			// Style should be available for installation
-			$parent = $cfg['parent'];
+			$parent = $style_data['extra']['parent-style'];
 			$style = array(
 				'style_id'			=> 0,
-				'style_name'		=> $cfg['name'],
-				'style_copyright'	=> $cfg['copyright'],
+				'style_name'		=> $style_data['name'],
+				'style_copyright'	=> $style_data['license'],
```

Comment:
The old copyright is not equivalent to the licence field in composer. The old copyright field detailed who had the copyright, this details the license (e.g. MIT)